plat/drivers/virtio/virtio_net.c

```diff
@@ -1022,16 +1040,25 @@ static int virtio_netdev_rxtx_alloc(struct virtio_net_device *vndev,
 	int vq_avail = 0;
 	int total_vqs = conf->nb_rx_queues + conf->nb_tx_queues;
 	uint16_t ctrl_size;
+	uint16_t vq_pairs;
 
 	__u16 qdesc_size[total_vqs];
 
-	if (conf->nb_rx_queues != 1 || conf->nb_tx_queues != 1) {
+	vq_pairs = conf->nb_rx_queues > conf->nb_tx_queues ? conf->nb_rx_queues :
```

Comment:
The "vq_pairs" can be set after the following if block that checks "conf->nb_rx_queues != conf->nb_tx_queues".

And given they are equal, based on the "conf->nb_rx_queues != conf->nb_tx_queues", why set to the maximum value one? For future use somehow?